src/service/PeopleServices.cpp

```diff
@@ -65,8 +65,16 @@ bool PeopleServices::logout(int user_id) {
     // Storage::setCurrentUser(People());
 
     //TODO: need to replace by database
-    cout << Storage::getCurrentUser()->getName() << "Log out successfully !" << std::endl;
-    return true;
+    People* people = PeopleDao::selectOnePeople(user_id);
+    if(people != nullptr){
+        cout << people->getName() << "Log out successfully !" << std::endl;
+        return true;
+    }else{
+        cout <<"The user ID: "<< user_id << "is not esist. "<< endl;
+        return false;
+    }
+    delete people;
```

Comment:
return statement will stop function immediately , so that people point wont delete forever.